tests/eve_tests/unit_tests/test_utils.py

```diff
@@ -72,7 +72,7 @@ class MyVirtualSubclassB:
         pass
 
     @eve.utils.register_subclasses(MyVirtualSubclassA, MyVirtualSubclassB)
-    class MyBaseClass(abc.ABC):
+    class MyBaseClass(abc.ABC):  # noqa: B024
```

Comment:
Wouldn’t it make more sense to either add an abstract method or remove the `abc.ABC`? That would make the test more realistic _and_ remove the requirement for `noqa`.

Reply:
We can't remove the `abc.ABC` inheritance because that is exactly what is being tested here. Adding an `abstractmethod` is ok and makes sense (although note that there are also some valid use cases for ABCs without abstractmethods).